### PR TITLE
Added 25 Angular & ML content

### PR DESCRIPTION
**Pull request by biswakpl**

### CODE REVIEW

**Comment by samtstern** [comment]
You've got an error:
```
x /home/runner/work/devlibrary/devlibrary/config/ml/repos/victordibia-handtrack.json is not valid!
  instance.repo: does not match pattern "^[^/]+$"
```

File: config/ml/repos/victordibia-handtrack.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "version": 1.0,
+    "source": "github",
+    "owner": "victordibia",
+    "repo": "handtrack.js/",
```
Comment (samtstern) [inline config/ml/repos/victordibia-handtrack.json:5]: This repo is not valid, I think it's the slash?

**Review by biswakpl — COMMENTED** [review]
Modified a repo name

**Review by samtstern — APPROVED** [review]

**Comment by samtstern** [comment]
I can backfill the authors for these. In the future using the `addproject` script will make this much simpler for you (and for me)

File: config/ml/repos/victordibia-handtrack.json (hunk 1 above)
Comment (biswakpl) [reply config/ml/repos/victordibia-handtrack.json]: yup

**Pull request merged**